config/defaults.json

```diff
@@ -0,0 +1,135 @@
+{
+  "cli_wallet_connection": "",
+  "db": {
+    "dialect": "sqlite",
+    "language": "en",
+    "storage": "/tmp/db.sqlite",
+    "sync": {
+      "force": true
+    }
+  },
+  "db-prod-example": {
```

Comment:
Documentation for people who want to see how it works with mysql. JSON does not allow comments.